test/sol5/kyberNetworkProxy.js

```diff
@@ -578,6 +578,41 @@ contract('KyberNetworkProxy', function(accounts) {
             });
         }
     });
+    
+    describe("test enable", async () => {
+        it("test set enable with admin", async () => {
+            await network.setEnable(false, { from: admin });
+            let enable = await networkProxy.enabled();
+            assert(!enable);
+            await network.setEnable(true, { from: admin });
+            enable = await networkProxy.enabled();
+            assert(enable);
+        });
+    });
+
+    describe("test swap function (backward compatible)", async () => {
+        it("swapEtherToToken", async () => {
+            let initialTakerBalance = await tokens[1].balanceOf(taker);
+            let tokenAdd = tokens[1].address;
+            let srcQty = (new BN(2)).mul((new BN(10)).pow(ethDecimals));
+            let proxyRate = await networkProxy.getExpectedRate(ethAddress, tokenAdd, srcQty);
+            Helper.assertEqual(proxyRate.worstRate, proxyRate.expectedRate.mul(new BN(97)).div(new BN(100)));
```

Comment:
getExpectedRate is backward compatible API
please create a separate test to this backward compatible API: getExpectedRate
and also one test for worst case value
something like:
"test backward compatible parameter worst rate" 

for any trade test, like this one, don't use the backward compatible API, prefer the new API